app/utils/refinementMappings.js

```diff
@@ -3,7 +3,7 @@ export const eligibilitiesMapping = {
   Families: ['Families', 'Families with Babies'],
   Homeless: ['Homeless'],
   Immigrants: ['Immigrants', 'Undocumented'],
-  LGBTQ: ['LGBTQ', 'Transgender'],
```

Comment:
`Transgender` is not currently an eligibility in the DB, so I've removed it. I've replaced it with `Transgender and Gender Non-Conforming`, which does exist in the DB. As part of this work, we plan to change the `LGBTQ` eligibility to `LGBTQ+`, so I've added that here